inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/support/WorkloadMetadataDialog.html

```diff
@@ -21,9 +21,22 @@
 <wicket:panel>
   <form>
     <div>
-      <div wicket:id="documentName"></div>
-      <div wicket:id="userInProgress"></div>
-      <div wicket:id="userFinished"></div>
+      <label>
+        <wicket:message key="DocumentName" />
+      </label>
+      <label wicket:id="documentName"></label>
+    </div>
+    <div>
+      <label>
+        <wicket:message key="InProgressUsers" />
+      </label>
+        <label wicket:id="userInProgress"></label>
+    </div>
+    <div>
+      <label>
+        <wicket:message key="FinishedUsers" />
+      </label>
+      <label wicket:id="userFinished"></label>
```

Comment:
The number of finished users is not a label - why is it in a `<label>` element? Best check comparable sites in the UI and how their HTML is coded.